features/support/ui/pages/money_manager.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../page'
+
+module UI::Pages
+  class MoneyManager < UI::Page
+    set_url '/en/tools/money-manager/'
+
+    element :received_first_payment_true,
+            '#questionnaire_received_first_payment_true'
+    element :received_first_payment_false,
+            '#questionnaire_received_first_payment_false'
```

Comment:
@igelstorm We are currently looking into whether to drop or uphold these standards. I do agree with you though